applications/fishing-map/src/features/workspace/common/Title.tsx

```diff
@@ -0,0 +1,30 @@
+import cx from 'classnames'
+import { UrlDataviewInstance } from '@globalfishingwatch/dataviews-client'
+import { useDataviewInstancesConnect } from '../workspace.hook'
+
+type TitleProps = {
+  dataview: UrlDataviewInstance
+  className: string
+  classNameActive: string
+  title: string
+}
+
+const Title = ({ dataview, className, classNameActive, title }: TitleProps) => {
```

Comment:
I had to wrap with `forwardRef` function to avoid `Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?`